src/components/cloud-cover/cloud-cover.scss

```diff
@@ -172,6 +180,18 @@ $_gap: ms.step(1);
       auto
       repeat(2, _cloud-space(0.5)); /* 1 */
   }
+
+  /**
+   * Modified styles for Cloud Cover with Extra content
+   *
+   * 1. Give the `extra` content a bit more space relative to the content
+   *    until the viewport is wide enough for the default evenly split layout.
+   */
+  .c-cloud-cover--has-extra & {
```

Comment:
While I think I _prefer_ `has-extra` in light of the `:has` selector, this goes against the convention of using `--with-` currently used by the Card and Heading components. For now we should probably make this `--with-extra` for consistency.